stix_shifter_modules/aws_athena/configuration/lang_en.json

```diff
@@ -0,0 +1,54 @@
+{
+    "connection": {
+        "help": {
+            "label": "Need additional help?",
+            "description": "More details on the datasource setting can be found in the specified link"
+        },
+        "region": {
+            "label": "Region",
+            "placeholder": "us-east-1",
+            "description": "Specify the region of the Amazon Athena for the data source"
+        },
+        "s3_bucket_location": {
+            "label": "S3 Bucket Location",
+            "placeholder": "s3://bucket_name/",
+            "description": "Specify S3 bucket location to save the results of query in s3 bucket"
+        },
+        "vpcflow_database_name": {
+            "label": "Database name for VPC Flow log",
+            "placeholder": "vpcflow_database_name",
+            "description": "Specify database name containing vpcflow service logs table from which the data to be queried"
+        },
+        "vpcflow_table_name": {
+            "label": "Table name for VPC Flow log",
+            "placeholder": "vpcflow_table_name",
+            "description": "Specify vpcflow service table name from which the data to be queried"
```

Comment:
Change to "Specify the name of the table containing the VPC Flow Logs"

Reply:
@delliott90 The description is updated with this change in the PR branch.